data/models/user-model.js

```diff
@@ -11,6 +11,15 @@ async function addNewUser(user) {
   }
 }
 
+async function findUsers() {
+  try {
+    const users = await db("users").select("id", "username", "email");
```

Comment:
Since you added an avatar_url field, maybe have that returned in the body of the response as well.